File: Java/1262. Greatest Sum Divisible by Three.java
class Solution {
    public int maxSumDivThree(int[] nums) {
        int[] arr = new int[3];
        for(int i: nums){
            int[] copy = arr.clone();
            for(int j: copy){
                arr[(i+j)%3]=Math.max(arr[(i+j)%3],i+j);
            }
        }
        return arr[0];
    }
}

File: C#/1262. Greatest Sum Divisible by Three.cs
public class Solution {
    public int MaxSumDivThree(int[] nums) {
        int[] arr = new int[3];
        foreach(int i in nums){
            int[] copy = (int[]) arr.Clone();
            foreach(int j in copy){
                arr[(i+j)%3]=Math.Max(arr[(i+j)%3],i+j);
            }
        }
        return arr[0];
    }
}
